sdk/servicebus/service-bus/test/track2.samples.spec.ts

```diff
@@ -0,0 +1,619 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import {
+  SessionConnections,
+  Message,
+  ContextWithSettlement as ContextWithSettlementMethods,
+} from "../src/track2/models";
+import { env } from "process";
```

Comment:
This fails in the browser. 

Either scope the test file for node 
or 
Do this instead.. `const env = isBrowser() ? (window as any).__env__ : process.env;`

Reply:
We have getEnvVars() already - I'll switch over.

Reply:
Fixed.